git_machete/client.py

```diff
@@ -578,10 +580,12 @@ def advance(self, *, branch: LocalBranchShortName, opt_yes: bool) -> None:
 
         def connected_with_green_edge(bd: LocalBranchShortName) -> bool:
             return bool(
-                not self.__is_merged_to_upstream(
-                    bd, opt_no_detect_squash_merges=False) and
+                not self.__is_merged_to_upstream(bd, opt_no_detect_squash_merges=False) and
                 self.__git.is_ancestor_or_equal(branch.full_name(), bd.full_name()) and
-                (self.__get_overridden_fork_point(bd) or self.__git.get_commit_hash_by_revision(branch) == self.fork_point(bd, use_overrides=False, opt_no_detect_squash_merges=False)))
+                (self.__get_overridden_fork_point(bd) or
+                 self.__git.get_commit_hash_by_revision(branch) == self.fork_point(bd,
+                                                                                   use_overrides=False,
+                                                                                   opt_no_detect_squash_merges=False)))
```

Comment:
Huh it looks awkward... maybe sth like:
```suggestion
                 self.__git.get_commit_hash_by_revision(branch) == \
                   self.fork_point(bd, use_overrides=False, opt_no_detect_squash_merges=False)))
```